### PR TITLE
[MM-32122] Add translation strings for shared channels slash commands

### PR DESCRIPTION
**Pull request by isacikgoz**
#### Summary
Added some translation strings

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-32122

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -1110,18 +1238,218 @@
...
+    "id": "api.command_share.channel_name.hint",
+    "translation": "[name] - defaults to channel name"
+  },
+  {
+    "id": "api.command_share.channel_purpose.help",
+    "translation": "Channel purpose provided to remote instances"
+  },
+  {
+    "id": "api.command_share.channel_purpose.hint",
+    "translation": ""
```
Comment (wiggin77) [inline i18n/en.json:1279]: Is this meant to be empty?

File: i18n/en.json [hunk 2]
```diff
@@ -2014,6 +2342,10 @@
     "id": "api.remote_cluster.service_not_enabled.app_error",
     "translation": "The remote cluster service is not enabled."
   },
+  {
+    "id": "api.remote_cluster.update.app_error",
+    "translation": ""
```
Comment (wiggin77) [inline i18n/en.json:2347]: Empty?

File: i18n/en.json [hunk 3]
```diff
@@ -1034,18 +1034,146 @@
...
+    "id": "api.command_remote.remove_remote_id.help",
+    "translation": "Id of remote cluster remove"
+  },
+  {
+    "id": "api.command_remote.service_disabled",
+    "translation": "Remote Cluster Service is disabled."
+  },
+  {
+    "id": "api.command_remote.service_not_enabled",
+    "translation": "Remote cluster service not enabled."
```
Comment (wiggin77) [inline i18n/en.json:1163]:
```suggestion
    "translation": "Remote Cluster service not enabled."
```

**Review by wiggin77 — CHANGES_REQUESTED** [review]
Fantastic work @isacikgoz!  Just a couple of questions and suggestion.

File: i18n/en.json (hunk 2 above)
Comment (isacikgoz) [reply i18n/en.json:2347]: I think it came from the feature branch. PR: #16768

File: app/slashcommands/command_remote.go [hunk 4]
```diff
@@ -129,7 +129,7 @@ func (rp *RemoteProvider) doInvite(a *app.App, args *model.CommandArgs, margs ma
 
 	rcSaved, appErr := a.AddRemoteCluster(rc)
 	if appErr != nil {
-		return responsef("Could not add remote cluster: %v", appErr)
```
Comment (agnivade) [inline app/slashcommands/command_remote.go:132]: I would suggest to use translation templates to pass variables, and avoid passing the formatting characters as part of the translation string itself. It looks a bit weird for a translator to see `%v` in the middle of a string. It leaks abstractions.

Something like this:
`T("tx_code", map[string]interface{}{"appErr": appErr.Error()})`

**Review by agnivade — CHANGES_REQUESTED** [review]
Wow, this is a tremendous amount of work!

I just have a suggestion to improve passing of params.

File: app/slashcommands/command_remote.go (hunk 4 above)
Comment (isacikgoz) [reply app/slashcommands/command_remote.go:132]: I see, seems fair.

File: i18n/en.json (hunk 2 above)
Comment (wiggin77) [inline i18n/en.json:2347]:
```suggestion
    "translation": "We encountered an error updating the remote cluster."
```

**Review by agnivade — APPROVED** [review]
Crazy amount of work there!

**Review by wiggin77 — APPROVED** [review]
Great work @isacikgoz!

**Pull request merged**